chart/templates/jupyter-statefulset.yaml

```diff
@@ -44,7 +44,7 @@ spec:
         {{- end }}
         livenessProbe:
           exec:
-            command: ['CMD-SHELL', 'curl -f http://localhost:8888/api || exit 1']
+            command: ['sh', '-c', 'curl -f http://localhost:8888/api || exit 1']
```

Comment:
Actually, since we're here, can we change this to `httpGet` instead of `exec`?
It should look like this:
```yaml
          livenessProbe:
            httpGet:
              path: /api
              port: 8888
```

Reply:
Replaced. Thanks for the suggestion.